binary/Pos/Binary/Class/Primitive.hs

```diff
@@ -55,14 +60,26 @@ import           Pos.Binary.Class.Core (Bi (..), enforceSize)
 -- The output is represented as a lazy 'BSL.ByteString' and is constructed
 -- incrementally.
 serialize :: Bi a => a -> BSL.ByteString
-serialize = CBOR.Write.toLazyByteString . encode
+serialize = serializeWith 1024 4096
```

Comment:
Any comment on these numbers?

Reply:
They seem to give good results. I didn't spend much time trying to tweak them though.

The first number is how many bytes should be allocated for the first buffer. The second is how much to allocate for subsequent buffers, if needed.

Reply:
Ok, I guess it's alright, but every magic constant needs a comment anyway.